RecoTracker/TkTrackingRegions/plugins/PointSeededTrackingRegionsProducer.h

```diff
@@ -62,10 +64,11 @@ class  PointSeededTrackingRegionsProducer : public TrackingRegionProducer
     else if (modeString == "VerticesSigma") m_mode = VERTICES_SIGMA;
     else  edm::LogError ("PointSeededTrackingRegionsProducer")<<"Unknown mode string: "<<modeString;
 
-    // basic inputs
-    edm::ParameterSet point_input = regPSet.getParameter<edm::ParameterSet>("point_input");
-    eta_input          = point_input.getParameter<double>("eta");
-    phi_input          = point_input.getParameter<double>("phi");
+    // basic inputsi
+    edm::ParameterSet points = regPSet.getParameter<edm::ParameterSet>("points");
+    etaPoints = points.getParameter<std::vector<double>>("eta");
+    phiPoints = points.getParameter<std::vector<double>>("phi");
```

Comment:
I liked the previous form of `points` more (vector of PSets with eta and phi attributes instead of two vectors of doubles). That doesn't prevent storing the eta and phi values as `vector<double>` members.

OTOH I don't want to delay this PR any further, and maybe after its integration it's not worth to change the configuration interface anymore (before next year at least).